Tests/Unit/LoraKeysManagerFacade/PreferredGatewayTest.cs

```diff
@@ -91,7 +91,6 @@ public async Task When_Calling_After_Delay_Should_Return_First_Gateway()
 
             var req2 = new FunctionBundlerRequest() { GatewayId = "gateway2", ClientFCntUp = staleFcntUp, Rssi = -90 };
             var pipeline2 = new FunctionBundlerPipelineExecuter(new IFunctionBundlerExecutionItem[] { this.preferredGatewayExecutionItem }, devEUI, req2);
-            var res2 = await this.preferredGatewayExecutionItem.ExecuteAsync(pipeline2);
```

Comment:
Not so sure if this is a safe change - should we consider discarding instead? The operation itself might have a side effect

Reply:
Ignored with 8fd7eb42de7b8974bc25bc5426b455fa768baf28